.travis.yml

```diff
@@ -110,7 +110,7 @@ script:
   # Generate documentation and run doctests
   - PYTHONPATH=$PWD make -C docs html doctest
   # Run javascript tests
-  - grunt test --verbose --force
+  - grunt test --verbose
```

Comment:
On Fri, 09 Sep 2016, Debanjum wrote:

>    In .travis.yml:

>  > @@ -110,7 +110,7 @@ script:
>  >    # Generate documentation and run doctests
>  >    - PYTHONPATH=$PWD make -C docs html doctest
>  >    # Run javascript tests
>  > -  - grunt test --verbose --force
>  > +  - grunt test --verbose

>    with the --force if the js_tests the test runs wouldn't pass unlike before
>    when it was just noted but didn't actually fail the travis/buildbot runs.

so, you mean it was ignoring failures in JS tests before but now
would properly fail if tests fail?